vms/platformvm/validators/manager.go

```diff
@@ -192,11 +172,7 @@ func (m *manager) GetValidatorSet(ctx context.Context, height uint64, subnetID i
 		}
 	}
 
-	// cache the validator set
-	validatorSetsCache.Put(height, vdrSet)
-
 	endTime := m.clk.Time()
-	m.metrics.IncValidatorSetsCreated()
```

Comment:
Should we keep the metric for the number of times we created a validator set here? Since these other two update gauges they won't give us a measure of the number of total calls that were performed, which may also be useful data.